universal-login-react/src/ui/Onboarding/Onboarding.tsx

```diff
@@ -41,12 +41,17 @@ export const Onboarding = (props: OnboardingProps) => {
   };
 
   const onCreateClick = async (ensName: string) => {
-    walletCreationService.onBalancePresent(() => modalService.hideModal());
-    const wallet = await walletCreationService.initiateCreationFlow(ensName);
-    modalService.hideModal();
-    props.onCreate && props.onCreate(wallet);
+    await walletCreationService.initiateCreationFlow(ensName);
   };
 
+  useEffect(() => {
+    setImmediate(async () => {
```

Comment:
This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.
https://developer.mozilla.org/en-US/docs/Web/API/Window/setImmediate

Reply:
Webpack should transpile it to something thats supported by the standard.